Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8238,10 +8251,23 @@ def get_whois(domain, normalized=None):
     return parse_raw_whois(raw_data, normalized=normalized, never_query_handles=False,
                            handle_server=server_list[-1])
 
+
 # Drops the mic disable-secrets-detection-end
 
+def get_domain_from_query(query):
+    # checks for largest matching suffix inside tlds dictionary
+    suffix_len = max([len(suffix) for suffix in tlds if query.endswith('.{}'.format(suffix))] or [0])
+    if suffix_len != 0:
+        suffix_len += 1
```

Comment:
why?

Reply:
To add the first dot before the TLD that was found. Should I add a comment and example inside the code?

Reply:
yes please